### ISSUE TITLE
De-triplication of CLD detector constructors / compact files in key4hep-stack

### ISSUE THREAD

**Issue by vvolkl** [HEP-FCC/FCCDetectors#11]
Some detector source / compact files are currently present in triplicate in the key4hep-stack - in DetFCCeeIDEA, DetFCCeeCLD and LCGEO. Since historically the CLD detector model was in LCGEO, I'd propose to keep the files there and only keep a master .xml in this repo that uses $LCGEO for includes.

**Comment by EmanuelPerez** [comment]
A cleaning has to be done, yes. If I remember well, the description in these several places is identical, although the xmls are not. One should be very careful and not use an inconsistent set of files.

**Comment by vvolkl** [comment]
Yes indeed - there is already a versioning scheme in LCGEO, so I would just move the xml files as they are here to `FCCee_o2_v02` or `FCCee_o3_v01`. The changes in the c++ are mostly about using the sensitive detectors from FCCSW I think - here it should be possible to add a configuration option to change the sensitive detector from Gaudi, similar to what is done in ddsim.
